examples/ephemeral-spark/hbase-cluster.tf

```diff
@@ -47,15 +47,15 @@ module "emr-hbase" {
   core_instance_on_demand_count   = 4
   # core_instance_spot_count    = 4
   # core_bid_price_as_percentage_of_on_demand_price = 100
```

Comment:
Let's just take out these commented values. Bid price doesn't do anything anymore and we don't recommend anyone use spot instances.

Reply:
It think it was 6000 for the tick and 12000 for timeout right? Where can we find those values?

Reply:
Found a [PR](https://github.com/Datatamer/terraform-aws-emr/pull/64/files#diff-185731c815ceccc6bd832279981f1f9bfda0eac607f096a8b6255a8949fd75cf) by you in the EMR repo. Ill use that. You still need to merge it though :)